app/exporters/manual_publishing_api_exporter.rb

```diff
@@ -1,11 +1,14 @@
 class ManualPublishingAPIExporter
+  UPDATE_TYPES = %w(minor major republish).freeze
```

Comment:
Added a new module included in all 3 exporters/publishers in b0447a559c8811594454188ffd3b7f4d5352f9b2